tsl/test/expected/transparent_decompression_queries.out

```diff
@@ -163,3 +163,40 @@ GROUP BY 2, 3;
 (27 rows)
 
 RESET enable_hashagg;
+-- test if volatile function quals are applied to compressed chunks
+CREATE FUNCTION check_equal_228( intval INTEGER) RETURNS BOOL 
+LANGUAGE PLPGSQL AS
+$BODY$
+DECLARE
+    retval BOOL;
+BEGIN
+   IF intval = 228 THEN RETURN TRUE;
+   ELSE RETURN FALSE;
+   END IF;
+END;
+$BODY$;
+SELECT * from test_chartab 
```

Comment:
What is the expected output? What is the output in the failure case prior to this fix? Would help to add a few notes about this in a comment so that one knows what to look for.

Reply:
added comments.